assets/src/adaptivity/scripting.ts

```diff
@@ -576,9 +576,14 @@ export const templatizeText = (
       }
     }
     let strValue = stateValue;
-    /* console.log({ strValue, typeOD: typeof stateValue }); */
-
-    if (Array.isArray(stateValue)) {
+    if (
+      vars.length === 1 &&
+      text === `{${vars[0]}}` &&
+      Array.isArray(stateValue) &&
+      typeof stateValue === 'object'
```

Comment:
I think if Array.isArray is true, then it's going to be object because arrays are objects